packages/ffe-cards-react/src/DetailListCard/DetailListCardItem.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import classNames from 'classnames';
+import { string, oneOfType, node, func } from 'prop-types';
+
+const DetailListCardItem = props => {
+    const {
+        className,
+        label,
+        value,
+        clickable,
+        onClick,
+        href,
+        ...rest
+    } = props;
+
+    const role = onClick ? 'button' : href ? 'link' : null;
+    return (
+        <div
+            role={role}
+            className={classNames(
+                'ffe-detail-list-card__item',
+                {
+                    'ffe-detail-list-card__item--clickable': role,
+                },
+                className,
+            )}
+            onClick={onClick}
+            href={href}
+            {...rest}
+        >
+            <dt className="ffe-detail-list-card__item-label">
+                {typeof label === 'string'
+                    ? label
+                    : React.cloneElement(label, { ...label.props })}
+            </dt>
+            <dd className="ffe-detail-list-card__item-value">
+                {typeof value === 'string'
+                    ? value
+                    : React.cloneElement(value, { ...value.props })}
+            </dd>
+        </div>
+    );
+};
+
+DetailListCardItem.propTypes = {
+    /** Additional classnames */
+    className: string,
+    /** Content of the label / left column */
+    label: oneOfType([node, string]),
+    /** Content of the value / right column */
+    value: oneOfType([node, string]),
+    /** onClick-function on row  - if set role is set to button*/
+    onClick: func,
+    /** href on row - if set role is set to link*/
```

Comment:
en href på en div blir galet.  Att laga en a istellet for div er dessverre ikke heller greit.
![image](https://github.com/SpareBank1/designsystem/assets/2248579/fff2a382-20ad-488b-ac75-690245cb69d0)

Kanske den som bruker komponenten må stoppa lenkarna i dt eller dd selv. onClick kan vell stå men ju den som bruker komponenten sitt ansvar att passa på en god del ting